src/config.js

```diff
@@ -241,12 +241,72 @@ const filecoin = {
   addedBy: 'Mostafa Balata',
 };
 
+const bnt = {
+  tokenContract: '0x1f573d6fb3f13d689ff844b4ce37794d79a7ff1c',
+  information: {
+    aliasName: 'Bancor Network',
+    website: 'https://www.bancor.network/',
+    logo: 'https://www.bancor.network/static/images/favicon.ico',
+  },
+  events: {
+    Contribution: {
+      args: {
+        tokens: '_amount',
```

Comment:
this is very wrong! please check event definition in Solidity

Reply:
Also this is clearly visible that number of tokens issued that is displayed is so different from totalSupply that it should raise suspicion